python/ray/util/collective/collective.py

```diff
@@ -150,6 +151,54 @@ def init_collective_group(world_size: int,
     _group_mgr.create_collective_group(backend, world_size, rank, group_name)
 
 
+def declare_collective_group(actors, group_options):
+    """
+    Declare a list of actors in a collective group with group options.
+
+    Note: This function should be called in a driver process.
+
+    Args:
+        actors (list): a list of actors to be set in a collective group.
+        group_options (dict): a dictionary that contains group_name(str),
+                              world_size(int), rank(list of int, e.g. [0,1]
+                              means the first actor is rank 0, and the second
+                              actor is rank 1), backend(str).
+    """
```

Comment:
```suggestion
    """Declare a list of actors as a collective group with group options.

    Note: This function should be called in a driver process.

    Args:
        actors (list): a list of actors to be set in a collective group.
        group_options (dict): Contains group_name(str), world_size(int), 
            rank(list of int, e.g. [0,1] means the first actor is 
            rank 0, and the second actor is rank 1), backend(str).
    """
```

Reply:
addressed this one. Other docstrings have been addressed in a previous PR (will rebase and reflect them.)